src/components/dictionaryConcepts/components/ViewMappingsModal.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ReactTable from 'react-table';
+import {
+  Button, Modal, ModalHeader, ModalBody, ModalFooter,
+} from 'reactstrap';
+
+class ViewMappingsModal extends Component {
```

Comment:
This can be a functional component.

Reply:
You are right. I have updated it, thanks for pointing it out.